### ISSUE TITLE
I heard no sound when I ran 16 bit games

### ISSUE THREAD

**Issue by OrHoshmand** [otya128/winevdm#793]
I played Pacman from Microsoft Return of Arcade and there is no sound.

**Comment by anonimuos** [comment]
Do you have RoA 1994 edition of RoA Anniversary Edition? I just tried AE. Am getting sound in all 4 games:  Pac-Man, Dig-Dug, Galaxian and Pole Position.

**Comment by OrHoshmand** [comment]
I think I used 1996 edition with these 4 games.

**Comment by anonimuos** [comment]
Installed the 1996 edition too. No problems with sound. Are you using the latest build as of today? In case you aren't, go here: https://ci.appveyor.com/project/otya128/winevdm/history then click the latest build -> This build is recommended -> Artifacts and download the zip and overwrite your OTVDM files (except OTVDM.INI you can skip or merge if you INI is customized).

**Comment by cracyc** [comment]
This is probably a dupe of https://github.com/otya128/winevdm/issues/390, IMO a new release is needed so people stop downloading the very old last one.

**Comment by anonimuos** [comment]
Yes reputation-wise there needs to be release. People are downloading the old release, finding that their app does not work, and giving up on this insanely great project. An updated release will make a lot more apps compatible.

**Comment by WinserverSupport** [comment]
it is a great project! Wildcat! Navigator runs but with a few interprocess comm I/O related stuff I am sure can be resolve.  It can also be securely related. I saw different security behavior with a CLI install versus a GUI installation. I haven't had time to get to it. Yes I did initially try to address but I didn't see any updated binaries without going through the whole headache of downloading, setting up and re-compiling and so fought. 
 
---
HLS

**Comment by OrHoshmand** [comment]
Nothing helped me, I still hear no sound and I can't download the latest build.

**Comment by Amillennialist** [comment]
> 
> 
> I played Pacman from Microsoft Return of Arcade and there is no sound.

Same here. Tried the latest recommended above and still no sound (Anniversary Edition). And Ms. Pac-Man doesn't start. Such a promising project too.

**Comment by cracyc** [comment]
I can't find the Anniversary Edition but AFAICT, ms pacman from that is the same as the one from revenge of arcade which is a win32 program and so not a bug in winevdm.  As for the sound in the rest since I can't repo the problem I'd need a trace to have any hope of solving it.

**Comment by anonimuos** [comment]
The installer for an up-to-date build of OTVDM is now here: http://www.columbia.edu/~em36/otvdm.html (Even if it says August, it's updated at the end of October). Please install that version and then see if the game gives issues.

On my PC, both Return of Arcade and Return of Arcade: Anniversary Edition games (Pac-Man, Dig-Dug, Galaxian and Pole Position) run fine with sound. No issues. Ms Pac Man is 32-bit so Windows runs it, not OTVDM.

Creating a trace is easy. Click New Issue on the top right of GitHub, then 'Get Started' and it will have the instructions to create a trace.
